docs/src/userguide/wrapping_CPlusPlus.rst

```diff
@@ -277,19 +162,25 @@ A convenient and safe place to do so is in the `__cinit__` and `__dealloc__` met
 which are guaranteed to be called exactly once upon creation and deletion of the Python
 instance.
 
-::
-
-    cdef class PyRectangle:
-        cdef Rectangle* c_rect      # hold a pointer to the C++ instance which we're wrapping
-        def __cinit__(self, int x0, int y0, int x1, int y1):
-            self.c_rect = new Rectangle(x0, y0, x1, y1)
-        def __dealloc__(self):
-            del self.c_rect
-        ...
+.. literalinclude:: ../../examples/userguide/wrapping_CPlusPlus/rect_ptr.pyx
 
 If you prefer giving the same name to the wrapper as the C++ class, see the
 section on :ref:`resolving naming conflicts <resolve-conflicts>`.
 
+Compilation and Importing
+=========================
+
+To compile a Cython module, it is necessary to have a :file:`setup.py` file:
+
+.. literalinclude:: ../../examples/userguide/wrapping_CPlusPlus/setup.py
+
+Run ``$python setup.py build_ext --inplace``
```

Comment:
Should be `$ python` (to suggest a shell command) and not `$python` (which suggests a shell variable).